tests/longevity-mv-si-2days.yaml

```diff
@@ -0,0 +1,81 @@
+test_duration: 2880
+stress_cmd: ["cassandra-stress user profile=/tmp/longevity_mv_profile.yaml ops'(insert=15,read1=1,read2=1,read3=1,read4=1,read5=1)' cl=QUORUM n=10000000 -port jmx=6868 -mode cql3 native -rate threads=10 -errors ignore",
```

Comment:
remove the -errors ignore
threads=10 is enough load?

Reply:
I put "-errors ignore" by Shlomi's request